maggy/util.py

```diff
@@ -234,3 +234,14 @@ def _validate_ml_id(app_id, run_id):
     if prev_app_id == app_id and int(prev_run_id) >= run_id:
         return app_id, (int(prev_run_id) + 1)
     return app_id, run_id
+
+
+def set_ml_id(app_id, run_id):
```

Comment:
can you replace all occurrences of `experiment_utils._set_ml_id` with this new one here.